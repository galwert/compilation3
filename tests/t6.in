int fakeFunc(int one, int two)
{
    return one + two;
}

void main()
{
    int x;
    int y;
    x = 8;
    y = 6;
    int z = x + y;
    int print = 5; // Should error here (pdf page 3 - NO SHADOWING OF ANY IDENTIFIER)
}