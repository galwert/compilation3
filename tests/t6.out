---end scope---
one INT -1
two INT -2
line 13: identifier print is already defined
